SalesTax/src/main/java/Checkout.java

```diff
@@ -0,0 +1,85 @@
+import java.text.DecimalFormat;
```

Comment:
Look at proposed changes in attachment.
[Checkout.txt](https://github.com/TGClements/tdd/files/4961395/Checkout.txt)